nginx/config.go

```diff
@@ -205,6 +205,11 @@ http {
 			allow 127.0.0.1;
 			deny all;
 		}
+    location /nginx_status {
+      stub_status on;
+      allow 127.0.0.1;
+      deny all;
+    }
```

Comment:
Formatting is a bitch, how do you keep it consistent while writing go? What editor are you using?